examples/README.md

```diff
@@ -0,0 +1,47 @@
+# Examples
+
+To help demonstrate the features and capabilities of `aframe-physics-system`
+the follow collection of examples that have been prepared.
```

Comment:
the following collection of examples has been prepared.

Reply:
Thanks for catching this 😊  I've fix this grammar in 79d3ada.